bin/cloudgov/test/s3fs

```diff
@@ -82,11 +101,37 @@ function testDrupalStreamWrapperPrivate {
 }
 
 function testDrupalStreamWrapperHTTPS {
-  echo "Expecting to read a SEMVER vX.Y.Z from $HTTPS_TEST_URL"
+  echo
+  echo "Expecting to read a SERVER vX.Y.Z from $HTTPS_TEST_URL"
```

Comment:
I think Dan really meant SEMVER: https://semver.org/ ($HTTPS_TEST_URL is a version.txt)